pay/lib/src/widgets/apple_pay_button.dart

```diff
@@ -64,4 +64,7 @@ class ApplePayButton extends PayButton {
 
   @override
   late final Widget _payButton = _applePayButton;
+
+  @override
+  final bool _returnsPaymentDataSynchronously = true;
```

Comment:
```suggestion
  final bool _isSynchronousPaymentData = true;
```

Reply:
Thanks for the proposal @ViniciusDeep.
The payment data is not synchronous itself, but the way it is being returned. Let's find something that defines what the variable holds very clearly (eg.: `_isPaymentDataReturnedSynchronously`, `_isPaymentDataStream`).

Reply:
I've opted for `_collectPaymentResultSynchronously`. Does that convey a clear message in your opinion?

Reply:
100% Thanks sir!